.github/workflows/cd.yml

```diff
@@ -72,9 +72,20 @@ jobs:
           echo "TXS_VERSION=${TXS_VERSION}">> $GITHUB_OUTPUT
           echo "GIT_VERSION=${GIT_VERSION}">> $GITHUB_OUTPUT
       
+      - name: Detect if codesign
+        run: |
+          # Nonexistent context property evaluates to an empty string.
+          # https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/contexts#about-contexts
+          if [ -z "${{ secrets.SIGNPATH_API_TOKEN }}" ]; then
+            echo "IF_CODESIGN=false" >> "$GITHUB_ENV"
+            echo "::notice file=.github/workflows/cd.yml,title=Codesign skipped::Secret "SIGNPATH_API_TOKEN" not set."
```

Comment:
A little styling improvement: the double quotes enclosing `SIGNPATH_API_TOKEN` should be escaped. Not something important.

`s/"SIGNPATH_API_TOKEN"/\"SIGNPATH_API_TOKEN\"/`

```diff
-echo "::notice file=.github/workflows/cd.yml,title=Codesign skipped::Secret "SIGNPATH_API_TOKEN" not set."
+echo "::notice file=.github/workflows/cd.yml,title=Codesign skipped::Secret \"SIGNPATH_API_TOKEN\" not set."
```